src/Microsoft.PowerShell.Commands.Utility/commands/utility/NewTemporaryFileCommand.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.PowerShell.Commands
     /// <summary>
     /// The implementation of the "New-TemporaryFile" cmdlet
     /// </summary>
-    [Cmdlet(VerbsCommon.New, "TemporaryFile", SupportsShouldProcess = true, HelpUri = "https://go.microsoft.com/fwlink/?LinkId=526726")]
+    [Cmdlet(VerbsCommon.New, "TemporaryFile", SupportsShouldProcess = true, HelpUri = "https://go.microsoft.com/fwlink/?LinkId=821836")]
```

Comment:
This like resolves to PowerShell 5.1 (https://docs.microsoft.com/en-us/powershell/module/Microsoft.PowerShell.Utility/New-TemporaryFile?view=powershell-5.1)  There is a PowerShell 6 page for this cmdlet (https://docs.microsoft.com/en-us/powershell/module/Microsoft.PowerShell.Utility/New-TemporaryFile?view=powershell-6)

Reply:
So the new fwlink points to 5.1, and IMO it should probably point to the version-agnostic link (for which we can change the default later). This way, when we update to 6, we can rev the default for *all* modules on docs.microsoft.com to 6.1, and then we don't have to change the fwlinks.

However, if we go that route, we need to make sure that 6 has distinct fwlinks from 5.1, and i'm not sure we've got that work fully understood or costed (that's a *lot* of fwlinks).